blocks/customers/customers.css

```diff
@@ -0,0 +1,100 @@
+.customers{
+    margin-top: 20px;
+    margin-bottom: 60px;
+    padding:0 10px;
+}
+.customers__image{
+    position: relative;
+    background-image: url('../../images/forCustomers_mobile.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height:643px ;
+    border-radius: 20px;
+}
+.customers__content{
```

Comment:
The layout isn't accurate at 1024px, looks like the button should stay placed beneath the paragraph for a bit longer. Doing so would also prevent the wrapping of the text on the button.